src/client/app/common/views/pages/page-editor/page-editor.vue

```diff
@@ -220,6 +220,10 @@ export default Vue.extend({
 
 	methods: {
 		save() {
+			if(this.name == ''){
```

Comment:
Recommended
--

```suggestion
			if (!this.name.length) {
```

Alternatives
--

```suggestion
			if (this.name.length == 0) {
```

```suggestion
			if (this.name == '') {
```